opm/flowdiagnostics/utility/AssembledConnections.cpp

```diff
@@ -256,9 +256,7 @@ CSR::accumulateRowEntries(const int               numRows,
     }
 
     // Note index range: 1..numRows inclusive.
-    for (decltype(this->ia_.size())
-             i = 1, numRows = this->ia_.size() - 1;
-         i <= numRows; ++i)
+    for (int i = 1; i <= numRows; ++i)
```

Comment:
You should really rename the upper bound, not change the type of the induction variable here.